Makefile

```diff
@@ -94,9 +94,10 @@ uninstall-emacs:
 
 install-lib:
 	install -d $(LIB_DIR)
+	cp -r std $(LIB_DIR)
 	install -m 644 prelude.m31 $(LIB_DIR)/prelude.m31
 uninstall-lib:
-	rm -f $(LIB_DIR)/prelude.m31
+	rm -rf $(LIB_DIR)/* || true
```

Comment:
I copied whatever was used for `examples`. I thought it was there with a purpose. I agree with switching to a saner build & install system.